src/stratis_cli/_actions/_pool.py

```diff
@@ -268,10 +267,11 @@ def init_cache(namespace):  # pylint: disable=too-many-locals
             ]
             raise StratisCliIncoherenceError(
                 (
-                    "Expected to add the specified blockdevs as cache "
-                    "to pool %s but stratisd reports that it did not actually "
-                    "add some or all of the blockdevs requested; devices "
-                    "added: (%s), devices requested: (%s)"
+                    f"Expected to add the specified blockdevs as cache to pool "
+                    f"{namespace.pool_name} but stratisd reports that it did not "
+                    f"actually add some or all of the blockdevs requested; "
+                    f"devices added: ({', '.join(devnodes_added)}), "
+                    f"devices requested: ({', '.join(blockdevs)})"
                 )
                 % (namespace.pool_name, ", ".join(devnodes_added), ", ".join(blockdevs))
```

Comment:
Missing deletion of the format line?

Reply:
Yes; sorry.  I thought I deleted that, but there must have been two before.